bedrock/contentful/utils.py

```diff
@@ -10,50 +10,36 @@
 from bedrock.contentful.models import ContentfulEntry
 
 
-def can_use_locale(
+# TODO? Cache this
+def locales_with_available_content(
     content_type: str,
     classification: str,
-    locale: str,
     default_locale: str = "en-US",
-) -> bool:
-    """Is there enough of this content type, of this classification,
-    in the given locale, compared to the default locale to, for example,
-    show a listing page of it?"""
+) -> List[str]:
+    """
+    Returns a list of locale names for which we have 'enough' content in that
+    locale to merit showing a listing page.
 
-    active_locales = get_active_locales(
-        content_type,
-        classification,
-        default_locale,
-    )
-    return locale in active_locales
+    Why? We don't want a resource center with only one article available in it;
+    better to wait a few days till translations for the rest of the content
+    lands in Bedrock.
 
-
-# TODO? Cache this
-def get_active_locales(
-    content_type: str,
-    classification: str,
-    default_locale: str = "en-US",
-    slug: str = None,
-) -> List:
-    """How many locales do we have 'enough' of to consider them an active locale?"""
+    Note that this is not about working how much of a _single_ Entry has been
+    localised - it's focused on the number of localised Entries in each
+    non-default locale compared to the number of entries in the default locale.
+    """
+    threshold_pc = settings.CONTENTFUL_LOCALE_SUFFICIENT_CONTENT_PERCENTAGE
 
     kwargs = dict(
         content_type=content_type,
         classification=classification,
+        localisation_complete=True,
     )
-    if slug:
-        kwargs["slug"] = slug
-
+    # Get all the locales _with at least one complete localisation each_
     all_locales = ContentfulEntry.objects.filter(**kwargs).values_list("locale", flat=True)
```

Comment:
Would this return dupes that may need to be de-duped to be unique?

Reply:
Thanks for the call-out. I think the dupes are/were deliberately there as part of a Counter-like thing that shows how much of each locale we have. I could look to take it out, but I think it might be handy for metrics on numbers of locales, too, so if it comes out it might go back in after a while :).